<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTest">
    <property name="title">
     <string>Test</string>
    </property>
    <addaction name="actionEditForms"/>
    <addaction name="actionLoadFormFromFile"/>
   </widget>
   <widget class="QMenu" name="menuGroup">
    <property name="title">
     <string>Group</string>
    </property>
    <addaction name="actionEditGroups"/>
    <addaction name="actionLoadGroupFromFile"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <widget class="QMenu" name="menuTesting">
    <property name="title">
     <string>Testing</string>
    </property>
    <addaction name="actionCreateFormsForGroup"/>
    <addaction name="separator"/>
    <addaction name="actionInputTestData"/>
    <addaction name="actionProcessing"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuTest"/>
   <addaction name="menuGroup"/>
   <addaction name="menuTesting"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="fingram.qrc">
     <normaloff>:/resources/settings.png</normaloff>:/resources/settings.png</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="fingram.qrc">
     <normaloff>:/resources/exit.png</normaloff>:/resources/exit.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="fingram.qrc">
     <normaloff>:/resources/help.png</normaloff>:/resources/help.png</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="fingram.qrc">
     <normaloff>:/resources/finlit.png</normaloff>:/resources/finlit.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="icon">
    <iconset resource="fingram.qrc">
     <normaloff>:/resources/qt.png</normaloff>:/resources/qt.png</iconset>
   </property>
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionEditForms">
   <property name="icon">
    <iconset resource="fingram.qrc">
     <normaloff>:/resources/editform.png</normaloff>:/resources/editform.png</iconset>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
   <property name="toolTip">
    <string>Edit questionnaires</string>
   </property>
  </action>
  <action name="actionLoadFormFromFile">
   <property name="text">
    <string>Load from text file</string>
   </property>
   <property name="toolTip">
    <string>Load group from text file</string>
   </property>
  </action>
  <action name="actionEditGroups">
   <property name="icon">
    <iconset resource="fingram.qrc">
     <normaloff>:/resources/editgroup.png</normaloff>:/resources/editgroup.png</iconset>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
   <property name="toolTip">
    <string>Edit groups</string>
   </property>
  </action>
  <action name="actionLoadGroupFromFile">
   <property name="text">
    <string>Load from text file</string>
   </property>
  </action>
  <action name="actionInputTestData">
   <property name="icon">
    <iconset resource="fingram.qrc">
     <normaloff>:/resources/edittesting.png</normaloff>:/resources/edittesting.png</iconset>
   </property>
   <property name="text">
    <string>Entering test data</string>
   </property>
  </action>
  <action name="actionCreateFormsForGroup">
   <property name="icon">
    <iconset resource="fingram.qrc">
     <normaloff>:/resources/crftesting.png</normaloff>:/resources/crftesting.png</iconset>
   </property>
   <property name="text">
    <string>Ð¡reate questionnaires for group</string>
   </property>
  </action>
  <action name="actionProcessing">
   <property name="icon">
    <iconset resource="fingram.qrc">
     <normaloff>:/resources/calculate.png</normaloff>:/resources/calculate.png</iconset>
   </property>
   <property name="text">
    <string>Test result processing</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="fingram.qrc"/>
 </resources>
 <connections/>
</ui>
